modules/contact/src/actions/ContactAction.C

```diff
@@ -742,19 +808,22 @@ ContactAction::addMortarContact()
     addMechanicalContactConstraints(normal_lagrange_multiplier_name,
                                     action_name + "_normal_constraint_",
                                     "NormalMortarMechanicalContact",
-                                    false);
+                                    false,
+                                    true);
```

Comment:
I would love this, but the lambda is defined not far away, so its really just a weak suggestion
```suggestion
                                    /* is_additional_frictional_constraint = */ false,
                                    /* is_normal_constraint = */ true);
```

Reply:
in fact, let's not make any change to PR right now